src/Consumer/AbstractMailerConsumer.php

```diff
@@ -42,6 +43,14 @@ protected function doExecute(array $data): int
             }
 
             return $delivered ? ConsumerInterface::MSG_ACK : ConsumerInterface::MSG_REJECT_REQUEUE;
+        } catch (ClientException $clientException) {
+            $this->writeln($data['uuid'], 'Client Exception');
+            $this->getLogger()->error(
+                'Error while sending a mail with UUID '.$data['uuid'],
```

Comment:
It would be nice to have the exception message in the log message too.

What do you think if we replace the beginning of the message by `Http client error` instead of `Error` ?